pages/docs/tutorials/streetlights.md

```diff
@@ -43,6 +43,7 @@ channels:
       summary: Inform about environmental lighting conditions for a particular streetlight.
       operationId: onLightMeasured
       message:
+        name: LightMeasured
```

Comment:
Please also add it in the second spec under line 151 :) In sending I have an error that message hasn't message name.

Reply:
![image](https://user-images.githubusercontent.com/20404945/113269100-69c45d80-92d8-11eb-9ac4-fc863154e062.png)

Reply:
yup, 🦅 👁️ 
Thanks! done